waspc/packages/wasp-config/__tests__/testFixtures.ts

```diff
@@ -71,731 +145,844 @@ type PageType = (typeof PAGE_TYPES)[number];
 const ENTITY_TYPES = ["task", "user", "social-user"] as const;
 type EntityType = (typeof ENTITY_TYPES)[number];
 
-export function createUserApp(scope: ConfigType): {
-  appName: string;
-  userApp: UserApi.App;
+export function createApp(scope: ConfigType): {
```

Comment:
I can't reach the line I want, but...

I think we should comment why `PAGE_TYPES` includes `CONFIG_TYPES`. It's my first time seeing that code and I don't really understand that.

Again, this is related to code that's not even shown in the diff, so we can skip it if you think it's too much.